adding D&I + EEO language

LauraUSDS opened the pull request:
- Adds D&I language to `/faq`
- Adds EEO language to bottom of `/apply`

Fixes #543

LauraUSDS committed:
Create 0101_david-holmes1.md

test for Laura

LauraUSDS committed:
Update 0101_david-holmes1.md

test 2 Laura

Commit:
remove test update file

LauraUSDS committed:
Adding D&I faq

Added an FAQ about USDS diversity & inclusion policy; answer is our EEO blurb.

LauraUSDS commented:
@drew-usds Could you pls double-check I added the D&I question correctly to the FAQ yml? This is for https://github.com/usds/website/issues/543

Also, would love help figuring out how to delete our fake Holmes commits from this PR....

Commit:
add eeo statement to /apply

Commit:
Merge branch 'staging' of https://github.com/usds/website into staging

drew-usds commented on _content/faq.yml:
```diff
@@ -27,10 +27,16 @@
       a: |-
         You should receive an email confirmation when your application is submitted. If you are selected to move on to interviews, you will be connected with a member of the USDS Talent Team. They will be your guide throughout your interview process.
 
+    - q: "How does USDS address diversity and inclusion?"
```
The EEO statement alone doesn't fully address what we do regarding diversity and inclusion. Someone could read this as the _only_ thing we do is _encourage_ people to apply. This headline should either be more specifically about EEO, or we could write another graph or two about how else we support diversity and inclusion.

LauraUSDS replied:
Agreed, good point. I've changed the wording of the question, and asked the team who wrote this language initially to consider making a second FAQ that is more about D&I in USDS culture / day-to-day operations.

drew-usds replied:
👍

LauraUSDS committed:
Edited D&I question wording

Current answer didn't fully answer what USDS does to encourage diversity & inclusion (agreed that should be there, but we're not there yet), so changing the wording of the question to more accurately reflect the info given.

Commit:
change eeo to eoe

Commit:
Merge branch 'staging' of https://github.com/usds/website into staging
